examples/seq2seq/seq2seq_trainer.py

```diff
@@ -35,6 +36,10 @@
 from transformers.training_args import ParallelMode
 
 
+if is_fairscale_available():
+    from fairscale.optim import OSS
```

Comment:
`OSS` is a bit cryptic to me, but I think it's still better to use the "real" name instead of `import OSS as OptimizerStateSharding` -> so good for me!

Reply:
Yeah I'm using the same convention they do too, to not surprise any user.